[2025-07-07T07:04:43.566+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-07T07:04:43.585+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: minio_ga4_new_file_detector.detect_new_ga4_data manual__2025-07-07T06:53:38.076557+00:00 [queued]>
[2025-07-07T07:04:43.589+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: minio_ga4_new_file_detector.detect_new_ga4_data manual__2025-07-07T06:53:38.076557+00:00 [queued]>
[2025-07-07T07:04:43.589+0000] {taskinstance.py:2303} INFO - Starting attempt 7 of 8
[2025-07-07T07:04:43.594+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): detect_new_ga4_data> on 2025-07-07 06:53:38.076557+00:00
[2025-07-07T07:04:43.599+0000] {standard_task_runner.py:63} INFO - Started process 3075 to run task
[2025-07-07T07:04:43.601+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'minio_ga4_new_file_detector', 'detect_new_ga4_data', 'manual__2025-07-07T06:53:38.076557+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/dag_automatic.py', '--cfg-path', '/tmp/tmp87x_pt42']
[2025-07-07T07:04:43.603+0000] {standard_task_runner.py:91} INFO - Job 146: Subtask detect_new_ga4_data
[2025-07-07T07:04:43.612+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T07:04:43.636+0000] {task_command.py:426} INFO - Running <TaskInstance: minio_ga4_new_file_detector.detect_new_ga4_data manual__2025-07-07T06:53:38.076557+00:00 [running]> on host 34b334fe6b50
[2025-07-07T07:04:43.686+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='minio_ga4_new_file_detector' AIRFLOW_CTX_TASK_ID='detect_new_ga4_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T06:53:38.076557+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T06:53:38.076557+00:00'
[2025-07-07T07:04:43.686+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-07T07:04:43.744+0000] {logging_mixin.py:188} INFO - B·∫Øt ƒë·∫ßu ki·ªÉm tra c√°c file m·ªõi h∆°n timestamp: 2025-07-07T06:11:42.444000+00:00
[2025-07-07T07:04:43.755+0000] {logging_mixin.py:188} INFO - üí§ Kh√¥ng t√¨m th·∫•y d·ªØ li·ªáu GA4 m·ªõi n√†o.
[2025-07-07T07:04:43.755+0000] {python.py:237} INFO - Done. Returned value was: False
[2025-07-07T07:04:43.755+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-07T07:04:43.770+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=minio_ga4_new_file_detector, task_id=detect_new_ga4_data, execution_date=20250707T065338, start_date=20250707T070443, end_date=20250707T070443
[2025-07-07T07:04:43.812+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-07T07:04:43.827+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-07T07:04:43.828+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
